From 989046a0aac8e7cb3db18e3cbc8a1aac82b82079 Mon Sep 17 00:00:00 2001
From: Matthias Brugger <mbrugger@suse.com>
Date: Tue, 16 Feb 2021 20:54:08 +0100
Subject: [PATCH] configs: RPi2: Disable EFI Grub workaround

The EFI Grub workaround isn't needed with Grub version 2.04 or higher.
This version was published over a year ago, so disable the workaround
to reduce boot time.

Signed-off-by: Matthias Brugger <mbrugger@suse.com>
---
 configs/rpi_2_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/rpi_2_defconfig b/configs/rpi_2_defconfig
index e546c3e7be..219c61b1f5 100644
--- a/configs/rpi_2_defconfig
+++ b/configs/rpi_2_defconfig
@@ -42,3 +42,4 @@ CONFIG_SYS_WHITE_ON_BLACK=y
 CONFIG_CONSOLE_SCROLL_LINES=10
 CONFIG_PHYS_TO_BUS=y
 CONFIG_OF_LIBFDT_OVERLAY=y
+# CONFIG_EFI_GRUB_ARM32_WORKAROUND is not set
